eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/Arbitrary.hs

```diff
@@ -107,23 +107,28 @@ import Test.Cardano.Ledger.Core.Arbitrary (
   genValidAndUnknownCostModels,
   genValidCostModel,
   genValidCostModels,
+  genericShrinkMemo,
  )
 import Test.Cardano.Ledger.Mary.Arbitrary ()
 import Test.Cardano.Ledger.Plutus (alwaysFailsPlutus, alwaysSucceedsPlutus)
 
 instance
   ( Arbitrary (AlonzoScript era)
   , AlonzoEraScript era
+  , NativeScript era ~ Timelock era
```

Comment:
```suggestion
  , Arbitrary (NativeScript era)
```

Reply:
It's a bit more complicated than this. The problem is that `AlonzoTxAuxData` holds a `Timelock`, not a `NativeScript`, so a lot needs to be changed to get rid of this type equality. I'll just keep it like that for now and create an issue to change the type of the field to `NativeScript`.

Reply:
Aha, you are right. That means we need to fix that (#5255). Feel free to ignore this suggestion for this PR